### PR TITLE
User can sort table by clicking on header. 

### PR DESCRIPTION
**Pull request by jyaaan**

### CODE REVIEW

**Review by victormejia — CHANGES_REQUESTED** [review]
in general, it's good practice to add the `.DS_Store` to the `.gitignore` file so that it's not added to git.

**Comment by jyaaan** [comment]
Ah, dang. I forgot to check with an ls -a.
I've removed the file.

File: main.js [hunk 1]
```diff
@@ -229,6 +235,24 @@ function updateMassLeads(leadIds, inputLead, masterLeads, massEditLead) {
 
 // LEAD DATA FUNCTIONS
 
+function sortLeads(leads, prop) {
+  var leadsCopy = leads;
```
Comment (thebearingedge) [inline main.js]: This isn't a real copy. Objects (including Arrays and Functions) are pass by reference. `Array.prototype.slice()` can be used to make a shallow copy.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by jyaaan** [comment]
Got it. changed the line to utilize slice()

**Pull request merged**